mod/aalborg_theme/views/default/aalborg_theme/css.php

```diff
@@ -67,6 +67,22 @@
 	margin-left: 6px;
 }
 /* ***************************************
+	ICONS
+*****************************************/
+.elgg-icon-hover-menu-hover,
+.elgg-icon-hover-menu,
+.elgg-icon-hover-menu:hover,
+:focus > .elgg-icon-hover-menu {
+	background: transparent url(<?= elgg_get_simplecache_url('aalborg_theme/avatar_menu_arrows.png') ?>) no-repeat;
```

Comment:
i think this was added so mobile users could click on the user icon, maybe @PerJensen can tell us if this is still needed?